config/routes.rb

```diff
@@ -1,9 +1,19 @@
 Rails.application.routes.draw do
-  get 'user/:user_id/posts' => 'posts#index', as: 'user_posts'
-  get 'users/:user_id/posts/:id' => 'posts#show', as: 'user_post'
-  get 'users' => 'users#index', as: 'users'
-  get 'users/:id' => 'users#show', as: 'user'
+  # get 'user/:user_id/posts' => 'posts#index', as: 'user_posts'
+  # get 'users/:user_id/posts/:id' => 'posts#show', as: 'user_post'
+  # get 'users' => 'users#index', as: 'users'
+  # get 'users/:id' => 'users#show', as: 'user'
```

Comment:
- [ ] OPTIONAL
There are commented code blocks found in your code on the repository. Kindly endeavor to remove **all** commented code from your code before pushing it to the repository so as to keep your code well organized.

Reply:
Thank you for the observation